### PR TITLE
feat: integrate swagger (XD-180)

### LINKED ISSUE #180: Integrate swagger open api
## User story

1. As developer
2. I want to have an well documented overview over all backend endpoints
3. So that I can integrate them into smoothly into the backend

## Acceptance criteria

- [x] The documentation should be based on swagger
- [x] Swagger should automatically generate and infer the types
- [x] Swagger should be integrated in its on lib.

## DoD general criteria

- [x] Feature has been fully implemented
- [x] Feature has been merged into the mainline
- [x] All acceptance criteria were met
- [x] Product owner approved features
- [x] All tests are passing
- [x] Developers agreed to release

### PR DESCRIPTION
**Pull request by KonsumGandalf**
Closes: #180

### CODE REVIEW

File: libs/common/backend/swagger/README.md
Comment (Persists) [inline libs/common/backend/swagger/README.md:1]: Add a short description

File: libs/facilities/backend/facilities/src/lib/controller/facilities.controller.ts
Comment (Persists) [inline libs/facilities/backend/facilities/src/lib/controller/facilities.controller.ts:1]: Add Api description for the facility endpoints?

**Review by Persists — CHANGES_REQUESTED** [review]

File: libs/common/backend/swagger/README.md
Comment (KonsumGandalf) [reply libs/common/backend/swagger/README.md:1]: Good point.

File: libs/facilities/backend/facilities/src/lib/controller/facilities.controller.ts
Comment (KonsumGandalf) [reply libs/facilities/backend/facilities/src/lib/controller/facilities.controller.ts:1]: Oh damn forgot that :( Thank you.

**Pull request merged**